odl/util/ufuncs.py

```diff
@@ -23,9 +23,8 @@
 is used to re-wrap the data into the appropriate space.
 """
 
-# Imports for common Python 2/3 codebase
 from __future__ import print_function, division, absolute_import
-
+from builtins import object
```

Comment:
Is this needed?

Reply:
Not always I guess, but the most critical case is when implementing `bool(my_obj)`. In Python 2 that would require `__nonzero__`, whereas in Python 3 and with `builtins.object` as base, the method must be `__bool__` just as we implement it. It also affects `__str__` with non-ASCII stuff (not relevant for us) and custom iterators (`next` vs. `__next__`).

Since quite some of our classes implement `bool`, I suggest we keep this as base class.

Reply:
I guess we can leave it, lets go!